### PR TITLE
APEXMALHAR-2397 #resolve Removing DAG.GATEWAY_CONNECT_ADDRESS which i…

### PR DESCRIPTION
**Pull request by sashadt**
…s causing evaluation failures during apex get-app-package-info call

### CODE REVIEW

**Comment by tweise** [comment]
PR would need to be rebased. 

How is the application supposed to work without the URI?

**Comment by sanjaypujare** [comment]
@sashadt  @tweise is right. You should only remove the check for dag.getValue(DAG.GATEWAY_CONNECT_ADDRESS) being null and throwing the RuntimeException since that causes problem in constructing the DAG in get-app-package-info command. The rest of the code is required for the app to work properly since the operators need the URI to be set. If the URI is not correct i.e. connectable then those operators will throw an exception in setup/activate and that ought to be enough for error-checking.

**Comment by sashadt** [comment]
@tweise There is no need to set the `uri` variable on `wsQuery` or `wsResult` because their internal implementation uses `uriHelper`, which sets this to a default value, which is correct.  Setting the `uri` is optional, and seems to be an override, which is not needed.

@sanjaypujare I'm not following your comment about "only remove the check for `dag.getValue(DAG.GATEWAY_CONNECT_ADDRESS)` ", because that is the key to getting the `uri` in the frist place.  If we don't make that call, do you have an alternative strategy for finding the correct `uri`?

In summary, I don't see a problem with this PR, and I don't see why `uri` needs to be set, if I don't want to provide an override, and am happy with the defaults.

**Comment by sanjaypujare** [comment]
@sashadt  my comment was based on @tweise 's comment. I did look at uriHelper based on your clarification and it does look like your changes should work so I have no issues.

**Comment by sanjaypujare** [comment]
@sashadt  you'll still need to rebase though...

**Comment by tweise** [comment]
@sashadt makes sense. Please rebase the PR.

**Comment by sashadt** [comment]
@tweise, @sanjaypujare Rebased.  Please merge.
